src/login/components/Inputs/CustomInput.js

```diff
@@ -5,13 +5,14 @@ import FormText from "reactstrap/lib/FormText";
 import Input from "reactstrap/lib/Input";
 import Label from "reactstrap/lib/Label";
 import i18n from "../../translation/InjectIntl_HOC_factory";
+import InputWithIcons from "./InputWithIcons";
```

Comment:
We are currently not using icon for showing/hide function. I recommend that we remove all icon from function/component names.